src/Accordion/Accordion.tsx

```diff
@@ -71,7 +72,7 @@ const Accordion: React.FunctionComponent<AccordionProps> = (props: AccordionProp
     }
 
     return (
-        <div className={"custom-accordion" + (props.className ? ` ${props.className}` : "")} id={props.id} >
+        <div className={"custom-accordion" + (props.className ? ` ${props.className}` : "") + (props.alternative ? " alternate-accordion" : "")} id={props.id} >
```

Comment:
Move this to a variable

```

let customClasses : string = "";
customClasses  = props.className ? props.className : "";

if(props.alterbative) {
    customClasses  += " " + props.alternative;
}

<div className={"custom-accordion" + customeClasses} id={props.id}>
``` 
Doing this would simplify things and ensure we don't keep elongating the className whenever we have another class to append.

Reply:
new state is introduced to cater for this